### PR TITLE
update README

### LINKED ISSUE #53: README needs to be updated
It was hastily written a long time ago--lots of room for improvement.  It should include the availability via conda-forge, and probably note that only the source tarball is available from PyPI.

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes #53

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,69 @@
...
 To test, after installation, run "pytest" from the source directory.
 
 ## Note for xarray users
 
-A wrapper around gsw called [gsw-xarray](https://github.com/DocOtak/gsw-xarray)
-exists for xarray. It adds CF compliant attributes when possible, units, and name.
+A wrapper around gsw called [gsw-xarray](https://github.com/DocOtak/gsw-xarray) exists for xarray.
+It adds CF compliant attributes when possible, units, and name.
 
 ## Note for on generating the docstrings
```
Comment (efiring) [inline README.md:66]: Delete "for"

File: README.md [hunk 2]
```diff
@@ -1,57 +1,69 @@
...
+
 The autogenerated docstrings are checked with codespell in the CIs.
-when autogenerating them the we need to run `pre-commit run --all-files` and fix the documentation issues found.
+when autogenerating them the we need to run ``pre-commit run --all-files`` and fix the documentation issues found.
```
Comment (efiring) [inline README.md:69]: Delete "the"

File: README.md [hunk 3]
```diff
@@ -1,57 +1,69 @@
...
+This Python implementation of the Thermodynamic Equation of Seawater 2010 (TEOS-10) is based primarily on numpy ufunc wrappers of the GSW-C implementation.
+This replaces the original [python-gsw](https://github.com/TEOS-10/python-gsw) pure-python implementation after a brief overlap period.
+The primary reasons for this change are that by building on the C implementation we reduce code duplication and we gain an immediate update to the 75-term equation.  
+Additional benefits include a major increase in speed,
+a reduction in memory usage,
+and the inclusion of more functions.
+The penalty is that a C (or MSVC C++ for Windows) compiler is required to build the package from source.
 
-**Warning: this is for Python >=3.5 only.**
+**Warning: this is for Python >=3.6 only.**
```
Comment (efiring) [inline README.md:16]: 3.8?

**Review by efiring — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ocefpaf) [reply README.md:16]: While I believe it can work on Python 3.6 you are right, we should mentioned only the versions that we are testing and packaging.

**Review by efiring — APPROVED** [review]

**Pull request merged**